nishiyama/chap1/knock03.py

```diff
@@ -0,0 +1,6 @@
+word = "Now I need a drink, alcoholic of course, after the heavy lectures involving quantum mechanics."

```

Comment:
```suggestion
word = "Now I need a drink, alcoholic of course, \
after the heavy lectures involving quantum mechanics."
```

使っているコードフォーマッターによって違いますが，
1行は79文字や88文字と決まっています（空白込み）．
このリポジトリでは，99文字までと設定しているので，99文字以内に1行が収まるように工夫してください．